runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -66,7 +67,8 @@ public Table(@Nonnull final String namespace,
                  @Nonnull final CorfuRuntime corfuRuntime,
                  @Nonnull final ISerializer serializer,
                  @Nonnull final Supplier<StreamingMap<K, V>> streamingMapSupplier,
-                 @NonNull final ICorfuVersionPolicy.VersionPolicy versionPolicy) {
+                 @NonNull final ICorfuVersionPolicy.VersionPolicy versionPolicy,
+                 final UUID[] streamTags) {
```

Comment:
Why does stream tag have to be 16 full bytes per tag? Seems super expensive given that the update itself might be just 4-8 bytes. Some table updates are pretty small but will need to be tagged in different stream tags. So say a 8 bytes of user data needs to be tagged in 3 stream tags, then user data will be 8 bytes but the overhead of just the tagging alone will be 48 bytes an amplification of 6 X !!

Reply:
sorry I realized that these are stream IDs, the only suggestion would be to use strings instead of UUIDs since verticals who consume deal with strings much more readily. thanks